spdx/parsers/jsonyamlxmlbuilders.py

```diff
@@ -159,6 +175,25 @@ class FileBuilder(rdfbuilders.FileBuilder):
     def __init__(self):
         super(FileBuilder, self).__init__()
 
+    def set_file_chksum(self, doc, chk_sum):
```

Comment:
this would make these names inconsistent with the rest.  Also there is a 'from spcx import checksum' in these files that causes more confusion than this refactor is worth.  IMHO.